test/iht/testhelpers/MessagesTidierTest.scala

```diff
@@ -618,14 +661,36 @@ class MessagesTidierTest extends UnitSpec with FakeIhtApp {
       }
     }
 
+    "compareMessageFileKeys" must {
+
+      "not fail when mocked english and welsh message files have the same keys" in {
+        val diff: Set[String] = mockedMessagesTidier.compareMessageFileKeys()
+        assert(diff == Set.empty)
+      }
+
+      "fail when mocked english and welsh message files do not have the same keys" in {
+        val diff: Set[String] = mockedMessagesTidier.compareMessageFileKeysWelshFailure
+        val diff2 = mockedMessagesTidier.compareMessageFileKeysEnglishFailure
+        assert(diff != Set.empty && diff2 != Set.empty)
+      }
+
+      "not fail with real messages files" in {
+        pending
+        val result = MessagesTidier.compareMessageFileKeys()
+        assert(result.isEmpty, "\n \n There are message keys missing from messages.en and/or messages.cy - " +
+          "see the file /home/" + System.getProperty("user.name") + "/Desktop/missingKeysAndValues.txt for more info")
```

Comment:
This won't work (I think) for Adam, as Desktop is as far as I'm aware only on ubuntu machines. Also it might be better to generate two files: one for missing in welsh one for missing in english. Or else save it in csv format so it can be opened with libreoffice easily.